libs/components/packages/src/schematics/rules/add-polyfills-config.ts

```diff
@@ -2,34 +2,43 @@ import { Rule } from '@angular-devkit/schematics';
 
 import { updateWorkspace } from '../utility/workspace';
 
-const polyfillsBuilders = [
```

Comment:
I removed the check for our builders for a few reasons:
1. For the `ng add` and `ng generate` schematics, since we're now requiring a `--project` be provided we don't need to insist on these builders when adding our polyfill.
2. For the `ng update` schematic, I adjusted the logic slightly to only add our polyfill to the workspace config if that project also included our `window.global` polyfill in `polyfill.ts` or `test.ts`. If the only check is the builder, then we might accidentally add our polyfill to a project that doesn't want it (because the default `@angular-devkit/build-angular:browser` could be applied to any project with-or-without SKY UX).

Reply:
That makes sense, but if the project is using `jest` then `polyfills` is not a valid option.

Reply:
Good catch. Updated.